library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -366,6 +373,23 @@ def update_etc_sysconfig_yast2
       def setup_snapper_quota
         Yast::Execute.on_target("/usr/bin/snapper", "--no-dbus", "setup-quota")
       end
+
+      # Parses the snapshots list
+      #
+      # @param content [String] Snapshots list content
+      # @return Array<Hash> An array of hashes containing the data for each snapshot
+      def parse_snapshots_list(content)
```

Comment:
this method is kind of tricky as any parsing method, I think that one or two example lines can make it more clear what happens in method.

Reply:
BTW, snapper offers a machine readable outuput, see http://snapper.io/2019/11/06/machine-readable-output.html.